src/zimscraperlib/zim.py

```diff
@@ -39,13 +42,23 @@ def __init__(
         self.name = name
         self.tags = tags
         self.scraper = scraper
+        self.source = source
+        self.flavour = flavour
+        self.redirects = redirects
```

Comment:
Redirects is a zimwriterfs feature (a path to a file). If should not be stored here and passed as arg to the other function.

Reply:
Okay. Will change that.